extcap_ot.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2019, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import os
+import sys
+import tempfile
+import argparse
+import subprocess
+import threading
+import logging
+import time
+import re
+
+from spinel.stream import StreamOpen
+from spinel.const import SPINEL
+from spinel.codec import WpanApi
+from serial.tools.list_ports import comports
+from enum import Enum
+
+# Nodeid is required to execute ot-ncp-ftd for its sim radio socket port.
+# This is maximum that works for MacOS.
+DEFAULT_NODEID = 34
+COMMON_BAUDRATE = [460800, 115200, 9600]
+
+class Config(Enum):
+    CHANNEL = 0
+    BAUDRATE = 1
+    TAP = 2
+
+def extcap_config(interface, option, extcap_version):
+    """List Configuration for the given interface"""
+    args = []
+    values = []
+    args.append((Config.CHANNEL.value, '--channel', 'Channel', 'IEEE 802.15.4 channel', 'selector', '{required=true}{default=11}'))
+
+    match = re.match(r'^(\d+)\.(\d+)$', extcap_version)
```

Comment:
Is it possible that `extcap_version` contains 3 digits: e.x. 3.0.1

Reply:
I have only seen '2.9' and '3.0' even if the actual Wireshark version contains 3 digits like '3.0.6'. But it does make sense to change regex to '^(\d+)(\.\d+)*$' since we only check the first digit.

Reply:
If you have checked this, I am OK with the current code